applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -25,7 +25,16 @@
 
 /**
  * Client that contains all the operations for {@link ConfigurationSetting ConfigurationSettings} in Azure Configuration
- * Store.
+ * Store. Operations allowed by the client are adding, retrieving, updating, and deleting ConfigurationSettings, and
+ * listing settings or revision of a setting based on a filter.
+ *
+ * <p><strong>Construct the client, for additional ways to build a client see {@link ConfigurationAsyncClientBuilder here}.</strong></p>
```

Comment:
I would changing this line. You should have only 'Instantiating an Asynchronous Configuration Client' or similar in bold. The rest of the text above should go in a new paragraph beneath the code block